.circleci/config.yml

```diff
@@ -1860,10 +1864,6 @@ workflows:
           <<: *on_develop
 
     jobs:
-      # OSSFUZZ builds and (regression) tests
-      - b_ubu_ossfuzz: *requires_nothing
```

Comment:
Why is this line removed?

Reply:
Just for testing. I'll restore it in a moment.

I mean, I was getting and error because of a duplicate `b_ubu_ossfuzz` in the workflow. Later I noticed that this is not because of this but another instance of it that's at the top of the `main` workflow. But it does not matter in this PR. `nightly` won't run here anyway.